src/bytecode/concrete.py

```diff
@@ -1143,6 +1143,17 @@ def concrete_instructions(self) -> None:
         free_instrs: List[int] = []
 
         for instr in self.bytecode:
+            if sys.version_info >= (3, 12) and isinstance(instr, Instr):
+                if instr.name == "LOAD_METHOD":
```

Comment:
Would there be a better place to handle these sorts of transformations? Just thinking of ways of keeping this tidy in case of future similar changes.